FirebaseRemoteConfig/Sources/RCNConfigDBManager.m

```diff
@@ -267,8 +267,8 @@ - (BOOL)createTableSchema {
 
   static const char *createTableMetadata =
       "create TABLE IF NOT EXISTS " RCNTableNameMetadata
-      " (_id INTEGER PRIMARY KEY, bundle_identifier"
-      " TEXT, fetch_time INTEGER, digest_per_ns BLOB, device_context BLOB, app_context BLOB, "
+      " (_id INTEGER PRIMARY KEY, bundle_identifier TEXT, namespace TEXT,"
```

Comment:
Changing table scheme is not a trivial change, we need to ensure backward compatible and ensure different version of Remote config can work with each other without crashing.

Reply:
As discussed we'll use a new table name for the new schema, so that a new metadata table will always be created when developers upgrade to this version or later

If they revert to an older version they can still read/write to the old table (which doesn't have a `namespace` column)